### DISCUSSION TITLE
Weird non functioning button on code blocks.

### DISCUSSION THREAD

**Discussion by champsblanc** [squidfunk/mkdocs-material#2749] [Q&A]
Wherever i add a codeblock there's a little grey square-shaped button appearing on the top left part of my blocks. I supposed it's related to the copy to clipboard functionality but it doesn't seem to do nothing (the copy button on the right does). I checked other docs using mkdcos and material theme but i can't see this square there. 

IDK if you will be able to see it in this screenshot but i highlighted the HTML line on the right.

![Screenshot from 2021-06-11 16-10-28](https://user-images.githubusercontent.com/61840925/121705737-3e7a9d80-cad5-11eb-8d06-f158d8552eaa.png) 

It's not a big problem but i'd like to not have it so it looks cleaner. Thanks!

**Comment by squidfunk** [comment]
Do you have customizations? The `input` doesn’t look as it should be there.

**Reply by champsblanc** [reply]
Like extra CSS? I only have this:

```
.md-grid {
  max-width: 1440px;
}

.md-content div{
  overflow-x: auto;
}
```

**Reply by squidfunk** [reply]
The official documentation doesn’t have the `input` field that seems to create this grey rectangle. The class `copy-code-button-doc` is nowhere used among the sources of Material for MkDocs.

Thus, it’s likely that the origin of the error is somewhere in your docs. If you can create a reproducible test case with a mint installation you may create an issue.
